[llgl] Add new port

NancyLi1013 opened the pull request:
Related issue #7538.

Commit:
[llgl] Add new port

Voskrese commented:
static on off

`-DLLGL_BUILD_STATIC_LIB=OFF`

```
vcpkg_configure_cmake(
    SOURCE_PATH ${SOURCE_PATH}
    PREFER_NINJA # Disable this option if project cannot be built with Ninja
    NO_CHARSET_FLAG # automatic templates

    OPTIONS 
      -DGaussLib_DISABLE_AUTO_INIT=OFF 
	  -DGaussLib_ASSERT_EXCEPTION=ON 
	  -DGaussLib_ENABLE_INVERSE_OPERATOR=ON 
	  -DGaussLib_ENABLE_SWIZZLE_OPERATOR=ON 
	  -DGaussLib_ROW_VECTORS=ON
	  -DGaussLib_ROW_MAJOR_STORAGE=ON
      -DGaussLib_REAL_DOUBLE=ON
#      -DGaussLib_INCLUDE_DIR:PATH=${CURRENT_INSTALLED_DIR}/include
      -DLLGL_BUILD_EXAMPLES=OFF
      -DLLGL_BUILD_RENDERER_DIRECT3D11=ON
      -DLLGL_BUILD_RENDERER_DIRECT3D12=OFF
      -DLLGL_BUILD_RENDERER_OPENGL=ON
      -DLLGL_BUILD_RENDERER_VULKAN=OFF
      -DLLGL_BUILD_STATIC_LIB=OFF
      -DLLGL_BUILD_TESTS=OFF
      -DLLGL_BUILD_WRAPPER_CSHARP=OFF
      -DLLGL_D3D11_ENABLE_FEATURELEVEL=Direct3D 11.0
      -DLLGL_ENABLE_CHECKED_CAST=OFF
#      -DLLGL_ENABLE_DEBUG_LAYER=ON
      -DLLGL_ENABLE_JIT_COMPILER=OFF
      -DLLGL_ENABLE_SPIRV_REFLECT=OFF
      -DLLGL_ENABLE_UTILITY=OFF
      -DLLGL_GL_ENABLE_DSA_EXT=ON
      -DLLGL_GL_ENABLE_EXT_PLACEHOLDERS=ON
      -DLLGL_GL_ENABLE_OPENGL2X=OFF
      -DLLGL_GL_ENABLE_VENDOR_EXT=ON
      -DLLGL_GL_INCLUDE_EXTERNAL=ON
      -DLLGL_VK_ENABLE_EXT=OFF
)
```

NancyLi1013 commented:
@Voskrese, thanks for your comments. I noticed that GaussianLib is now only required for the Examples and Test projects in https://github.com/LukasBanana/LLGL/blob/ac7c6506521fe377f3fa4c80a580c064038c6520/docu/GettingStarted/Getting%20Started%20with%20LLGL.tex. So it should have no effects on other projects without GaussianLib.

Commit:
Add features and fix arm64 build error

NancyLi1013 commented:
@Rastaban, sorry for the late reply. I have fixed the error on arm64. Now this port doesn't currently support UWP and I have added this message in portfile.cmake.
